samples/facetime/app/src/main/java/com/cloudflare/facetime/sample/MeetingConfig.kt

```diff
@@ -0,0 +1,6 @@
+package com.cloudflare.facetime.sample
+
+internal object MeetingConfig {
```

Comment:
```suggestion
object MeetingConfig {
```

Not really relevant for application modules since nothing depends on them.